### DISCUSSION TITLE
DevOps add max-async-wait-time to PowerPlatformRestoreEnvironment

### DISCUSSION THREAD

**Discussion by arjanterheegde** [microsoft/powerplatform-build-tools#340] [Ideas]
Hello, 

Is it possible to add max async wait time to the PowerPlatformRestoreEnvironment task?
If i want to restore our production enviroment to test it will time out because it's needing more then 1 hour (default) 

![image](https://user-images.githubusercontent.com/6838586/206435182-0ca328a4-5937-49ad-96dd-2e18233b4305.png)
![image](https://user-images.githubusercontent.com/6838586/206434716-36a25065-d5da-4b28-9d41-68803b7d40f6.png)

**Comment by beumah** [comment]
Second this. We have the same issue with CopyEnvironment.

**Comment by tehcrashxor** [comment]
Added to our backlog, internal tracking as [AB#3306126](https://dev.azure.com/dynamicscrm/1fb98997-2f9e-4734-be8a-9728680447c2/_workitems/edit/3306126)
